### ISSUE TITLE
Support `tracks.conf` in addition to `trackList.json` in JBrowseController

### ISSUE THREAD

**Issue by deepakunni3** [GMOD/Apollo#1710]
Currently if there is any track configuration specified in `tracks.conf` in a data directory, those tracks wont be served by JBrowseController.

Although `trackList.json` is ubiquitous, people do configure their tracks in `tracks.conf`.

**Comment by nathandunn** [comment]
@deepakunni3  You're working this, right? 

One request.   If the file is not there, can we return an empty string (or the equivalent) instead of a 404.   In general, we should only have 404s for things we *expect* to be there, but are not.  We could log a warning or info on the back if dev / admins need to find that data.

**Comment by deepakunni3** [comment]
Yes, thats how it is being implemented. If the file is not there, its just `log.info` without any other side-effects.

**Comment by deepakunni3** [comment]
@nathandunn Fixed via #1723
